blocks/bricks/interfaces.py

```diff
@@ -141,31 +141,61 @@ class Initializable(RNGMixin, Brick):
 
     """
     has_biases = True
```

Comment:
backward compatibility. I know there was a discussion about dealing with biases like rest of parameters but it didn't look as it was conclusive. I can get rid of it, and treat backward compatibility with a replication warning.